CHANGES.rst

```diff
@@ -1,6 +1,34 @@
 Changelog
 =========
 
+Version 9.0.0
+~~~~~~~~~~~~~
+
+Nilupul Manodya implemented SAML 2.0 (Security Assertion Markup Language) Authentication.
+MSColab can now be configured with an existing IdP or multiple IdPs.
+In this way, a user can authenticate themselves in one system and gain access to another system
+by providing proof of authentication. In our documentation in the Components section you find a detailed description.
+
+Matthias Riße refactored the testing frame work and optimized and accelerated our tests for parallel testing.
+Various fixtures are used in the process.
```

Comment:
```suggestion
Matthias Riße refactored the test suite and optimized and accelerated our CI test runs.
All tests run in parallel now and are not retried upon failure, and most tests are also executed in a randomized order, instilling more confidence in the results.
Quite a bit of duplicated code was unified, mostly using pytest fixtures.
Additionally, CI test runs now also happen on x86_64- and ARM-based macOS.
```

Reply:
Do we really need to mention people by name here?

Reply:
I am in favor of explicitly mentioning people in the changelog, but in the spirit of being able to do faster releases I would prefer if the changelog was auto-generated from the information that already exists in git.